core/app/models/concerns/spree/user_methods.rb

```diff
@@ -0,0 +1,43 @@
+module Spree
+  module UserMethods
+    extend ActiveSupport::Concern
+
+    include Spree::UserApiAuthentication
+    include Spree::UserPaymentSource
+    include Spree::UserReporting
+
+    included do
+      has_many :role_users, class_name: 'Spree::RoleUser', foreign_key: :user_id
+      has_many :spree_roles, through: :role_users, class_name: 'Spree::Role', source: :role
+
+      has_many :promotion_rule_users, class_name: 'Spree::PromotionRuleUser'
+      has_many :promotion_rules, through: :promotion_rule_users, class_name: 'Spree::PromotionRule'
+
+      has_many :orders, foreign_key: :user_id, class_name: "Spree::Order"
+
+      belongs_to :ship_address, class_name: 'Spree::Address'
+      belongs_to :bill_address, class_name: 'Spree::Address'
+    end
+
+    def self.ransackable_associations(auth_object=nil)
+      %w[bill_address ship_address]
+    end
+
+    def self.ransackable_attributes(auth_object=nil)
```

Comment:
Unused method argument - `auth_object`. If it's necessary, use `_` or `_auth_object` as an argument name to indicate that it won't be used. You can also write as `ransackable_attributes(*)` if you want the method to accept any arguments but don't care about them.<br>Surrounding space missing in default value assignment.